exporter/elasticsearchexporter/config.go

```diff
@@ -273,9 +274,8 @@ func (cfg *Config) Validate() error {
 		return fmt.Errorf("unknown mapping mode %q", cfg.Mapping.Mode)
 	}
 
-	if cfg.Compression != "" {
-		// TODO support confighttp.ClientConfig.Compression
-		return errors.New("compression is not currently configurable")
+	if cfg.Compression != "none" && cfg.Compression != configcompression.TypeGzip {
```

Comment:
`TypeNone` should work? https://github.com/open-telemetry/opentelemetry-collector/blob/v0.111.0/config/configcompression/compressiontype.go#L17
```suggestion
	if cfg.Compression != configcompression.TypeNone && cfg.Compression != configcompression.TypeGzip {
```

Reply:
We should also probably treat empty value "" (`TypeEmpty`) as valid.

Reply:
notice it is lowercase `typeNone`, not `TypeNone`. And because it is not exported, it will not be possible to be referenced from es exporter.

Reply:
> We should also probably treat empty value "" (TypeEmpty) as valid.

With a non-empty default, I don't think it is possible to set it to empty.